src/icons.scss

```diff
@@ -687,4 +687,27 @@
     @extend .fa;
     @extend .fa-refresh;
   }
+
+  // Misc
+
+  &-search {
+    @extend .fa;
+    @extend .fa-fw;
+    @extend .fa-search;
+  }
+
+  // Alias
```

Comment:
IMHO aliases are not pertinent, maybe the menu icons should be replaced with these new ones.